### PR TITLE
update ADOPTERS.md to add KubeSphere as an adopter for KubeVirt

### PR DESCRIPTION
**Pull request by liuzhen21**
**What this PR does / why we need it**:
update ADOPTERS.md to add KubeSphere as an adopter for KubeVirt 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

```release-note
KubeSphere added to the adopter's file!
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 | Vendor | EQUINIX | | [link](https://metal.equinix.com/) | |
 | Vendor | H3C | 2019 | [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) | We distribute KubeVirt as part of CloudOS to enable VM workloads on Kubernetes at customer sites. |
 | Vendor | KUBERMATIC | 2019 | [link](https://www.kubermatic.com/products/kubevirt/) | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. |
+| Vendor | KUBESPHERE | 2020 | [link](https://kubesphere.com.cn/ksv/) | KubeSphere Virtualization(KSV) provides the lightweight VM management capability based on KubeVirt. |
```
Comment (aburdenthehand) [inline ADOPTERS.md:25]:
```suggestion
| Vendor | KUBESPHERE | 2020 | [link](https://kubesphere.com) | KubeSphere Virtualization (KSV) provides lightweight VM management capability based on KubeVirt. |
```

**Review by aburdenthehand — COMMENTED** [review]
Hello @liuzhen21, and thank you for the PR! Always happy to see new entries in our adopters file. 
Just to let you know I added a release note to your PR description.
I've also suggested to change the url suffix

File: ADOPTERS.md (hunk 1 above)
Comment (calvinyv) [reply ADOPTERS.md:25]: thanks @aburdenthehand for your quick response, We don't own that .com domain as our main site, so could we use https://kubesphere.cloud/en/ksv/ as an alternative?

Comment (aburdenthehand) [reply ADOPTERS.md]: Ah, I see. Yes, that makes sense. Thanks!

**Pull request merged**